docs/community/meetings/2024-05-09.md

```diff
@@ -0,0 +1,201 @@
+# 9 May 2024
+
+## Presentation by T.K. Le
+
+In this meeting [T.K. Le](https://github.com/thaopletk) gave a presentation about a series of COVID-19 modelling projects and how their experiences were impacted by the choice of programming languages, model structure, editors, tools, etc.
```

Comment:
It should be "TK Le" (TK is kind of not an abbreviation)

Reply:
Fixed. Sorry TK, I don't know why I wrote it as initials, but once I did, I did so throughout. I've resolved each instance.